scripts/docker-summary.sh

```diff
@@ -3,14 +3,17 @@ set -euo pipefail
 
 DOCKER_IDS=$(docker ps -aq)
 
+mkdir -p docker-info
+
 for id in ${DOCKER_IDS}
 do
   echo "***********************************************************"
   echo "***************Docker Container ${id}***************"
   echo "***********************************************************"
-  docker ps -af id=${id} --no-trunc
-  echo "----"
-  docker logs ${id} | tail -n 10 || echo "It is not possible to grab the logs of ${id}"
+  docker ps -af id="${id}" --no-trunc
+  echo "---- docker logs ----"
+  docker logs "${id}" | tail -n 10 || echo "It is not possible to grab the logs of ${id}"
+  docker inspect "${id}" > docker-info/"${id}"-docker-inspect.json
 done
 
```

Comment:
```suggestion
The Docker stasts ara also nice to have
```suggestion
  echo "***********************************************************"
  echo "***************Docker Stats***************"
  echo "***********************************************************"
  docker stats --no-trunc --no-stream
```
```